apps/src/code-studio/components/AssetManager.jsx

```diff
@@ -248,8 +258,9 @@ export default class AssetManager extends React.Component {
           {...this.defaultAssetProps(asset)}
           api={boundApi}
           onChoose={() => console.log('choose!')}
-          onDelete={() => console.log('delete!')}
+          onDelete={() => this.deleteStarterAssetRow(asset.filename)}
           levelName={this.props.levelName}
+          hideDelete={!this.props.isStartMode}
```

Comment:
this looks great, I like how it keeps the AssetRow agnostic of what kind of asset it contains.